submissions/akinbobolaFaith/phpMySqlDatabase/customersList.php

```diff
@@ -0,0 +1,41 @@
+<?php
+require_once("cookie.php");
+
+if (isset($_COOKIE["customers"])) {
+    $data = json_decode($_COOKIE["customers"], true);
+    } else {
+    echo "Sorry...Not Recognized";
+}
+
+try {
+    if($result -> rowCount() > 0){
```

Comment:
The `$result` is from the query.  Meaning anytime there is a page reload. It always hit the database.